tests/integration/suite.go

```diff
@@ -0,0 +1,121 @@
+package integration
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/suite"
+
+	contracts "github.com/smartcontractkit/ccip-owner-contracts/gethwrappers"
+	"github.com/smartcontractkit/timelock-worker/tests/containers"
+)
+
+type integrationTestSuite struct {
```

Comment:
the integration tests lifecycle and utilities methods are implemented using testify's suite package.